### PR TITLE
docs: Add faq about breaking change

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   -        |
| 🇨🇳 Chinese |        -   |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by MadCcc — APPROVED** [review]

File: docs/react/faq.zh-CN.md [hunk 1]
```diff
@@ -49,6 +49,18 @@ title: FAQ
...
+## 如何避免升级导致的破坏性变更？
+
+antd 在大版本中会避免引入破坏性变更，遵从以下原则会确保不会破坏你的代码：
+
+- 出现在官方 Demo 中的写法
+- FAQ 中出现的解法，包含代码片段以及 codesandbox 示例、issue 中当前版本标记 FAQ label 的
+
+而下述变更则需要开发者自行校验：
+
+- 特定场景的错误用法，明显在任意变动都不会生效的（例如 Tabs 下直接包 div 的用法）
```
Comment (afc163) [inline docs/react/faq.zh-CN.md:61]: > 明显在任意变动都不会生效的

这句话有点看不懂。

Comment (zombieJ) [reply docs/react/faq.zh-CN.md:61]: 那和英文统一好了：bug as feature

File: docs/react/faq.zh-CN.md [hunk 2]
```diff
@@ -49,6 +49,18 @@ title: FAQ
 
 你可以覆盖它们的样式，但是我们不推荐这么做。antd 是一系列 React 组件，但同样是一套设计规范。
 
+## 如何避免升级导致的破坏性变更？
+
+antd 在大版本中会避免引入破坏性变更，遵从以下原则会确保不会破坏你的代码：
```
Comment (afc163) [inline docs/react/faq.zh-CN.md:54]: 定义什么是『大版本』？

Comment (zombieJ) [reply docs/react/faq.zh-CN.md:54]: major

**Pull request merged**